arch/x86/core/crt0.S

```diff
@@ -91,9 +91,10 @@ SECTION_FUNC(TEXT_START, __start)
 	movw	$DATA_SEG, %ax	/* data segment selector (entry = 3) */
 	movw	%ax, %ds	/* set DS */
 	movw	%ax, %es	/* set ES */
-	movw	%ax, %fs	/* set FS */
-	movw	%ax, %gs	/* set GS */
 	movw	%ax, %ss	/* set SS */
+	movw	$0, %ax
```

Comment:
`xorw %ax, %ax` is 1 byte shorter.

Reply:
yeah the assembler doesn't like that, I tried. I don't think xor will take segment registers as operands

Reply:
disregard, looking at the wrong thing